readthedocs/builds/views.py

```diff
@@ -63,7 +65,18 @@ def post(self, request, project_slug):
             slug=version_slug,
         )
 
-        _, build = trigger_build(project=project, version=version)
+        update_docs_task, build = trigger_build(project=project, version=version)
+        if (update_docs_task, build) == (None, None):
+            # Build was skipped
+            messages.add_message(
+                request,
+                messages.WARNING,
+                "This project is currently disabled and can't trigger new builds.",
```

Comment:
Seems like this would happen a large number of times for a skipped project. Is this de-duped at all?

Reply:
This only happens when the build is manually triggered and we are using regular Django message system which adds the message into the request and the message is consumed immediately in the next response.

So, the flow is:

1. project is mark as skip
1. user click on `Build version` button
1. user is redirected to Build List page
1. this message appears

Next time the user visits any different page, this message won't be there anymore.

Reply:
ah ok. So this won't display when builds are triggered via GH and other options?

Reply:
No. When triggered via webhook I'm returning a 406 status code and a detailed message with the explanation of the situation.